src/components/Onboarding/OnboardingScreen.js

```diff
@@ -0,0 +1,668 @@
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  pageButtons: {
+    marginTop: '50%',
+  },
+  backButton: {
+    marginRight: theme.spacing(1),
+  },
+  instructions: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+  underlinedSubtitleText: {
+    textDecoration: 'underline',
+    textDecorationColor: '#53AA48',
+    fontWeight: 'bold',
+    marginBottom: '2%',
+
+  },
+  subtitleText: {
+    textAlign: 'left',
+    fontSize: '30px',
+    fontWeight: 'bold',
+    marginBottom: '7%',
+  },
+  regSubtitleText: {
+    textAlign: 'left',
+    paddingLeft: '10%',
+    fontSize: '30px',
+    marginBottom: '2%',
+  },
+  cenSubtitleText: {
+    textAlign: 'center',
+    fontSize: '30px',
+    fontWeight: 'bold',
+    marginBottom: '7%',
+  },
+  cenRegSubtitleText: {
+    textAlign: 'center',
+    paddingLeft: '10%',
+    fontSize: '30px',
+    marginBottom: '2%',
+  },
+  labelText: {
+    fontWeight: 'bold',
+  },
+  grayButton: {
+    backgroundColor: '#5e5e5e',
+    color: '#fff',
+    '&:hover': {
+      backgroundColor: '#5e5e5e',
+    },
+  },
+  blackButton: {
+    backgroundColor: '#373737',
+    color: '#fff',
+    '&:hover': {
+      backgroundColor: '#373737',
+    },
+    marginTop: '10%',
+  },
+  greenButton: {
+    backgroundColor: '#53AA48',
+    '&:hover': {
+      backgroundColor: '#53AA48',
+    },
+    marginTop: '10%',
+  },
+  submitButton: {
+    backgroundColor: '#53AA48',
+    '&:hover': {
+      backgroundColor: '#53AA48',
+    },
+    width: '50%',
+  },
+  textBox: {
+    width: '70%',
+  },
+  card: {
+    margin: 16,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+  },
+  stepper: {
+    margin: 'auto',
+    width: '60%',
+  },
+  icon: {
+    color: 'red !important',
+  },
+});
+const ROLE_TITLES = ['buyer', 'nonprofit', 'seller'];
+const BUTTONS = [
+  {
+    id: 0,
+    title: 'Buyer',
+    image: Fruit3,
+    styling: useStyles.fruit3,
+    fruit: {
+      marginTop: '10%',
+
+      height: '134px',
+      width: '98px',
+      left: '89px',
+      top: '46px',
+    },
+  },
+  {
+    id: 1,
+    title: 'Non-profit or Agency',
+    image: Fruit2,
+    styling: useStyles.fruit2,
+
+    fruit: {
+      marginTop: '10%',
+      height: '130px',
+      width: '105px',
+      left: '96px',
+      top: '50px',
+    },
+  },
+  {
+    id: 2,
+    title: 'Seller',
+    image: Fruit1,
+    styling: useStyles.fruit1,
+
+    fruit: {
+      marginTop: '10%',
+      height: '141px',
+      width: '91px',
+      left: '72px',
+      top: '39px',
+      // border-radius: '0px',
+    },
+  },
+];
+
+const INITIAL_FORM_STATE = {
+  firstName: '',
+  lastName: '',
+  email: '',
+  password: '',
+  confirmPassword: '',
+  org: '',
+  phone: '',
+  zipcode: '',
+  comments: '',
+};
+
+function getSteps() {
+  return ['Step 1', 'Step 2', 'Step 3', 'Confirmation'];
+}
+
+export default function OnboardingScreen() {
```

Comment:
Question in general, can we implement a popup so that way when the user presses next at step 3, there is a confirmation message? Don't worry about implementing this now, I just messaged the designers. For now, put a TODO comment in your code to make note of that